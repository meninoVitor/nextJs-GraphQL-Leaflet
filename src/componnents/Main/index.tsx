const Main = () => (
  <div>
    <h2>React avanÃ§ado</h2>
  </div>
)

export default Main
