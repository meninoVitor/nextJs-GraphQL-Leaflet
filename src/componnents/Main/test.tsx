import { screen, render } from '@testing-library/react'

import Main from './index'

describe('<Main>React avançado</Main>', () => {
  it('should render a header', () => {
    render(<Main>React avançado</Main>)
    expect(
      screen.getByRole('heading', { name: 'React avançado' })
    ).toBeInTheDocument()
  })
})
