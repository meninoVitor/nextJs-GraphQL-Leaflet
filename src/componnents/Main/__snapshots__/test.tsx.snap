// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main>React avançado</Main> should render a header 1`] = `
<div
  class="styles__Wrapper-sc-azxw3i-0 hvfBUy"
>
  <img
    alt="Imagem de um átomo e ao lado React Avançado"
    class="styles__Logo-sc-azxw3i-1 pytIO"
    src="/img/logo.svg"
  />
  <h1
    class="styles__Title-sc-azxw3i-2 dervBO"
  >
    React Avançado
  </h1>
  <h2
    class="styles__Description-sc-azxw3i-3 uRmVH"
  >
    TypeScript, ReactJS, NextJS e Styled Components
  </h2>
  <img
    alt="Pessoa em frente ao computador programando"
    class="styles__Ilustration-sc-azxw3i-4 iSGLcC"
    src="/img/hero-illustration.svg"
  />
</div>
`
